dev/archery/archery/utils/lint.py

```diff
@@ -120,8 +118,79 @@ def python_linter(src):
                                      check=False))
 
 
+def python_numpydoc(whitelist=None, blacklist=None):
+    """Run numpydoc linter on python.
+
+    Pyarrow must be available for import.
+    """
+    logger.info("Running python docstring linters")
+    packages = {
+        'pyarrow',
+        'pyarrow.dataset',
```

Comment:
Add `compute`, `csv`, `feather`, `ipc`, `json` and `types` ?

Reply:
Yes, I need to add these modules as well.